example/integration_tests/bazel6-components/sample/Main.java

```diff
@@ -0,0 +1,17 @@
+import org.graalvm.polyglot.Context;
+
+import java.io.BufferedOutputStream;
+import java.io.OutputStream;
+
+public class Main {
+    public static void main(String args[]) {
+        OutputStream myOut = new BufferedOutputStream(System.out);
+        Context context = Context.newBuilder("js")
+                .out(myOut)
+//                .option("js.Strict", "true")
+                .allowAllAccess(true)
+                .build();
+        context.eval("js", "42");
+        context.close();
```

Comment:
I see how you are using `js`. **Here's what you may want to do:**

```starlark
graalvm_repository(
    name = "graalvm",
    components = [
        "js",  # you only need this, not the `setup_action`
    ],
    distribution = "oracle",
    java_version = "20",
    version = "20.0.2",
)
```

**Then, in your `BUILD.bazel` file:**
```starlark
native_image(
    name = "main-native",
    main_class = "Main",
    native_image_tool = "@graalvm//:native-image",
    deps = [":java"],
    extra_args = [
        "--language:js",
    ],
)
```

This will build JS into your native image target. If you are looking to use JS in the `java_binary`, you will need to set GraalVM as your Java toolchain, so it runs it with your GraalVM installation.